services/QuillLMS/engines/evidence/app/models/evidence/hint.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Evidence
+  class Hint < ApplicationRecord
+    belongs_to :rule, inverse_of: :hint
+
+    validates_presence_of :explanation
+    validates_presence_of :image_link
```

Comment:
I suggest calling it `image_name` or something, since `link` could be construed as a fully qualified url. URLs are best managed by the app, not persisted in the DB (so that staging and prod can have full separate environments, or if we need to change the image URL we don't want to backfill the DB).

Tangentially: this PR uses the `Image` class for uploading the images. Have you considered adding the relation `Hint has_one Image`?

Reply:
image_link is, as it stands at the moment, a fully qualified url. I see what you're saying about the benefits of having the app handle the actual linking, but I think if we were going to build out a different system here we'd probably also want to address the other places we use the Image class for image uploads, and that feels outside the scope of this project. The other thing is that from talking to Hannah and Lindsey, my understanding is that they wanted this hint system in place quickly and that it would probably get iterated on further later to not use the static images, so I don't want to do too much fiddling with it here.